docs/cow-protocol/reference/core/auctions/competition_rules.md

```diff
@@ -0,0 +1,48 @@
+---
+id: competition-rules
+sidebar_position: 2
+---
+
+# Solver competition rules
+
+All solvers participating in the solver competition must abide by certain rules. In this section, we outline all these rules, which are naturally split into three classes: (1) the ones enforced explicitly by the smart contract, (2) the ones enforced by the offchain protocol infrastructure, and (3) the so-called social consensus rules.
+
+## Rules enforced by the smart contract
+
+- Limit price constraint: this rule enforces that an order cannot be executed if its limit price is violated.
+
+## Rules enforced by the offchain protocol infrastructure
+
+- Uniform clearing prices (UCP): this rule is an integral component of the protocol, and specifies that there is one clearing price per token per batch.
+
+- A solution is valid only if it contains at least one user order.
+
+- Every solution is associated with a score, and the solutions are ranked in decreasing order of their scores. The empty solution with the zero score is always considered as a candidate solution. The solver whose solution has the highest (positive) score is declared the winner of the batch auction, and gets the right to execute its solution onchain. The solver that provided the winning solution is then rewarded according to the rules specified in [CIP-20](https://snapshot.org/#/cow.eth/proposal/0x2d3f9bd1ea72dca84b03e97dda3efc1f4a42a772c54bd2037e8b62e7d09a491f).
```

Comment:
The sentence on the empty solution always confuses me a bit. Is this understandable for people not intimately familiar with the mechanism? @mfw78 
Dropping the sentence as well as the parentheses around "positive" would be clearer to me.

Reply:
Yes, agreed. I pushed a change that implements your suggestion.